packages/larva-tokens/src/brands/wwd.json

```diff
@@ -13,23 +13,23 @@
 			"type": "color",
 			"category": "text-color"
 		},
-		"FONT_FAMILY_PRIMARY_FANCY": {
+		"FONT_FAMILY_BODY": {
 			"value": "{!HELVETICA_NEUE}",
 			"type": "font-family",
 			"category": "font-family"
 		},
-		"FONT_FAMILY_SECONDARY_FANCY": {
+		"FONT_FAMILY_PRIMARY": {
```

Comment:
@sarahsmo I think you do need to have the fancy versions otherwise Papyrus will show 😬 - see the built versions

Reply:
oh hahahh! ok good catch thanks!